brownfield_django/templates/main/team/team_home.html

```diff
@@ -32,27 +30,36 @@
     </div>
       
     {% block course_documents %}
-    
     <div class="course-div">
-        <h3>Course Documents</h3>
+      <h3>Course Documents</h3>
         <div class="course-docs">
-                    {% for document in document_list %}
-            <li>{{ document.name }}
-                {% if document.name == "Link: Brownfield Action Reference Site" %}
-                    <a class="btn btn-default pull-right" href=http://brownfieldref.ccnmtl.columbia.edu/" target="_blank">View</a></li>
-                {% else %}
-                     <a class="btn btn-default pull-right" href="../../..{{STATIC_URL}}{{ document.link }}" target="_blank">View</a></li>
-                {% endif %}
-            {% empty %}
-            <li>Sorry, no documents have been made available yet.</li>
-            {% endfor %}
-            </div>
+          {% for document in document_list %}
+            <li>
+              {{ document.name }}
+              {% if document.name == "Link: Brownfield Action Reference Site" %}
+                <a class="btn btn-default pull-right" href=http://brownfieldref.ccnmtl.columbia.edu/" target="_blank">View</a>
+                </li>
+              {% else %}
+                <a class="btn btn-default pull-right" href="../../..{{STATIC_URL}}{{ document.link }}" target="_blank">View</a>
```

Comment:
Why `"../../..{{STATIC_URL}}`? That will definitely break when/if it moves to S3/Cloudfront.